Library/Homebrew/lock_file.rb

```diff
@@ -35,3 +35,15 @@ def create_lockfile
     @lockfile.fcntl(Fcntl::F_SETFD, Fcntl::FD_CLOEXEC)
   end
 end
+
+class FormulaLock < LockFile
+  def initialize(name)
+    super("#{name}.formula")
```

Comment:
I'm not sure I can see/understand why this change is needed? Can you explain a little more?